examples/gno.land/r/boards/public.gno

```diff
@@ -61,7 +61,7 @@ func CreateReply(bid BoardID, threadid, postid PostID, body string) PostID {
 	caller := std.GetOrigCaller()
 	if usernameOf(caller) == "" {
 		if !checkAnonFee() {
-			panic("payment must not be less than " + strconv.Itoa(gDefaultAnonFee))
+			panic("please register, other minimum fee required if anonymous " + strconv.Itoa(gDefaultAnonFee))
```

Comment:
```suggestion
			panic("please register, otherwise minimum fee required if anonymous " + strconv.Itoa(gDefaultAnonFee))
```

Reply:
Thank you, good catch, I will make a new PR since there are additional testing cases added.